src/PrMerger.js

```diff
@@ -278,7 +278,7 @@ class PrMerger {
 } // PrMerger
 
 // promises to process all PRs once, hiding PrMerger from callers
-async function Step(prIds) {
+export default async function Step(prIds) {
```

Comment:
```suggestion
export async function Step(prIds) {
```

Please apply similar changes to other cases that can be converted without heroic efforts.

Reply:
Ok. I left only Config as default - to avoid Config.Config everywhere...